packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -283,7 +284,11 @@ export const removeItemFromCart = payload => {
             }
         }
 
-        await dispatch(getCartDetails({ forceRefresh: true }));
+        if (!lastItemCount) {
```

Comment:
Move comment here. I would also suggest flipping this logic to make it more readable. Suggested block:

```js
// When removing the last item in the cart, perform a reset of the Cart ID
// to prevent a bug where the next item added to the cart has a price of 0.
// Otherwise refresh cart details to get updated totals.
if (isLastItem) {
    await clearCartId();
} else {
    await dispatch(getCartDetails({ forceRefresh: true }));
}
```